src/Microsoft.Tye.Hosting/Model/V1/V1Metric.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Microsoft.Tye.Hosting.Model.V1
```

Comment:
Nit: add the standard file license header to each new file.  (It looks like there are a couple of existing files that don't have it, which we should fix at some point, but is outside the scope of this PR.)

Reply:
Fixed